### PR TITLE
Corrected Prop 1.1 + Typos

### PR DESCRIPTION
**Pull request by shura11**
Copy/Paste of my commit "Corrected Prop 1.1" :

iii) was the definition of ii), there is nothing to prove.
The second part of the proof doesn't make sense in my opinion, so I replaced it with something similar to what we did last year (which correspond to the actual Lemma 1.24). Why is it weird ? It doesn't follow immediately from the definition of non degeneration that for all non zero vector x, x^T.A is not zero. That's exactly what we need to prove.

### CODE REVIEW

**Comment by EisenIn** [comment]
Hi Dylan, are you sure that iii) was the definition of ii)? We have non-degeneracy from the right and non-degeneracy from the left. The form might be non-symmetric. The idea of this proposition was that the direction of degeneracy does not matter, if the vector space is of finite dimension. 

Also, I would say that it follows from the definition of non-degeneracy that for all non-zero x, x^T A is non-zero. Otherwise we would have with [u]_B=x,  <u,v>=0 for each v in V. Maybe I could be more explicit, but I am not entirely unhappy with my proof. 

Bests,F.

**Comment by EisenIn** [comment]
But finally, your proof is clearer. I Merge the pull request

**Comment by EisenIn** [comment]
But now the part iii) of Prop. 1 is gone. Do you see that it makes sense? If yes, could you re-insert it?

File: chapter01-produits-scalaires.tex [hunk 1]
```diff
@@ -115,21 +119,19 @@ \chapter{Formes bilinéaires}
   \begin{enumerate}[i)]
   \item $\rank(A_B^f) = n$ \label{item:12}
   \item $f$ est non dégénérée \label{item:13}
-  \item Si $v \in V$, et si $\pscal{w,v}=0$ pour tout $w \in V$, alors $v = 0$. \label{item:14}
```
Comment (EisenIn) [inline chapter01-produits-scalaires.tex]: This needs to be re-inserted.

File: chapter01-produits-scalaires.tex [hunk 2]
```diff
@@ -115,21 +119,19 @@ \chapter{Formes bilinéaires}
...
   \end{enumerate}
 \end{proposition}
 
-\begin{proof}
-  Nous montrons \ref{item:12}) et \ref{item:13}) sont équivalentes.  Du même manière, on démontre aussi que \ref{item:12}) et \ref{item:14}) sont équivalentes.
```
Comment (EisenIn) [inline chapter01-produits-scalaires.tex]: This needs to be re-inserted.

**Comment by EisenIn** [comment]
I re-inserted the parts related to iii). Let me know if you still have doubts about its relevance. Thanks, F.

**Comment by shura11** [comment]
Ahh, I didn't see that v and w were now switched. You are absolutely right. My bad.
As for you proof, I got it now as well (I don't remember why it wasn't that clear before). I think we can keep it.
Thanks for pointing this out !

**Comment by shura11** [comment]
By the way, I don't really know how reviews work here on GitHub but it seems it didn't change anything and has the sole purpose of commenting changes only. I'll correct this.

**Pull request merged**